tests/testthat/test-stat_stepribbon.R

```diff
@@ -0,0 +1,130 @@
+#' @title Specifications test-stat_stepribbon.R
+#' @section Last updated by: Daniel Sjoberg (danield.sjoberg@@gmail.com)
+#' @section Last update date: 2022-07-12T04:47:32
+#'
+#' @section List of tested specifications
+#' T1. visR::stat_stepribbon() accepts aes() args ymin and ymax.
+#' T1.1 visR::stat_stepribbon() inherits aes from ggplot()
+#' T1.2 visR::stat_stepribbon() respects aes from visR::stat_stepribbon()
```

Comment:
what do you mean with respects?

Reply:
can you use 'inherits'?

Reply:
i am not sure what you're asking. can you try using what youre proposing with the function?

Reply:
Still dont understand what this test is doing. Perhaps you can resolve this with Mark during my time off.